client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigationInfoBar.tsx

```diff
@@ -57,7 +59,34 @@ const InvestigationInfoBar: React.FC<Props> = ({ currentTab }: Props) => {
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
 
-    React.useEffect(() => { 
+    const noInvestigationError = () => {
+        Swal.fire({
+            icon: 'warning',
+            title: 'נכנסת לעמוד חקירה מבלי לעבור בדף הנחיתה! הנך מועבר לשם',
+            customClass: {
+                title: classes.swalTitle
+            },
+            timer: 1750,
+            showConfirmButton: false
+        });
+
+        timeout(1900).then(() => history.push(landingPageRoute));
+    }
+
+    React.useEffect(() => {
+        timeout(2000).then(() => {
```

Comment:
why timeout is needed here ?

Reply:
the package that i used (redux state sync) updates each tab when opened, depending on the previous tabs state. to do so it dispatches certain actions after the tab has been loaded. this timeout is to ensure that the state sync has finished before checking which investigation was chosen or if there is need to go back to landing page. if you think you have a better solution im more than happy to hear it.